### DISCUSSION TITLE
Welcome to Distributed-Internet-Gateway Discussions!

### DISCUSSION THREAD

**Discussion by MichaelTaylor3d** [Datalayer-Storage/Distributed-Internet-Gateway#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hemasecret** [comment]
This is Jack from Chia in China. We are running a Chia community - HemaDAO
![HemaDAO 450 A](https://github.com/Datalayer-Storage/Distributed-Internet-Gateway/assets/88307252/649e99bc-a46d-4b0b-a240-1fc49f469a92)
 in China main land.

**Reply by dkackman** [reply]
Welcome Jack!

**Comment by Hemasecret** [comment]
How to set up the DIG node? 
Is there a diagram to explain the full concept about datalayer and applications on it?

**Reply by dkackman** [reply]
> How to set up the DIG node?

What operating system are you on? 

> Is there a diagram to explain the full concept about datalayer and applications on it?

not quite yet

**Comment by Hemasecret** [comment]
windows

---- Replied Message ----
| From | Don ***@***.***> |
| Date | 03/11/2024 11:36 |
| To | Datalayer-Storage/Distributed-Internet-Gateway ***@***.***> |
| Cc | Hemasecret ***@***.***>,
Comment ***@***.***> |
| Subject | Re: [Datalayer-Storage/Distributed-Internet-Gateway] Welcome to Distributed-Internet-Gateway Discussions! (Discussion #72) |

How to set up the DIG node?

What operating system are you on?

Is there a diagram to explain the full concept about datalayer and applications on it?

not quite yet
